### DISCUSSION TITLE
Where can I find the "vertical exaggeration" setting?

### DISCUSSION THREAD

**Discussion by fillwithjoy1** [osmandapp/OsmAnd#19774] [Q&A]
I cannot find this setting https://osmand.net/blog/osmand-android-4-7-released/#vertical-exaggeration

This is what I see in my terrarin settings. And yes I'm running OSMAnd 4.7
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/29514446/692a441c-8713-4e5c-a72b-fee869c68468"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by vshcherb** [comment]
It should be in 4.7.10 / 4.7.15 - released yesterday going to be 100% coming days

**Reply by fillwithjoy1** [reply]
I'm using version 4.7.10 btw. Is it under a hidden setting I haven't noticed?
